project5/3700kvstore

```diff
@@ -1,18 +1,22 @@
 #!/usr/bin/python3 -u
 import argparse
-import socket
-import select
-import time
+import enum
 import json
+import logging
+import math
+import os
 import random
+import select
+import socket
 import threading
-import math
-from enum import Enum
+import time
+
+import myutils
 
 # ----------------------------- GLOBAL VARIABLES -----------------------------
 
 MAX_BYTES = 32768 # maximum bytes to read in from the socket
-HEARTBEAT_FREQ = 0.5 # seconds, or 500 milliseconds
+HEARTBEAT_FREQ = 0.1 # seconds, or 500 milliseconds
```

Comment:
Need to change the comment to say 100 milliseconds (unless you were planning on changing it back to 0.5)

Reply:
Right

This time needs be less than the election minimum timeout time, because if it's not then there's no point to heartbeats

Reply:
Ah right right.